library/src/main/java/com/airbnb/android/airmapview/GoogleChinaWebViewMapFragment.java

```diff
@@ -1,9 +1,12 @@
 package com.airbnb.android.airmapview;
 
 public class GoogleChinaWebViewMapFragment extends WebViewMapFragment {
-
   public static GoogleChinaWebViewMapFragment newInstance(AirMapType mapType) {
     return (GoogleChinaWebViewMapFragment) new GoogleChinaWebViewMapFragment()
         .setArguments(mapType);
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
it is not implemented in `WebViewMapFragment`, only in `GoogleWebViewMapFragment`